index.html

```diff
@@ -4,38 +4,75 @@
 		<title>Matt W-R Portfolio</title>
 		<link rel="preconnect" href="https://fonts.gstatic.com" />
 		<link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Orbitron:wght@600&display=swap" rel="stylesheet" />
+		<link href="https://fonts.googleapis.com/css2?family=Libre+Barcode+39&display=swap" rel="stylesheet" />
 		<link rel="stylesheet" type="text/css" href="normalize.css" />
 		<link rel="stylesheet" type="text/css" href="styles.css" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	</head>
 	<body>
 		<nav>
 			<div class="nav-name">
-					<h2>Matt<span>hew</span> W<span>atling</span>-R<span>ead</span></h2>
+					<h2 tabindex="1">Matt<span>hew</span> W<span>atling</span>-R<span>ead</span></h2>
 			</div>
 			<div class = "nav-links">
-				<a href="#about-me"><span class="long-text">about me</span><span class="short-text">am</span></a>
-				<a href="#portfolio"><span class="long-text">portfolio</span><span class="short-text">p</span></a>
-				<a href="#contact"><span class="long-text">contact</span><span class="short-text">c</span></a>
+				<a href="#about-me" tabindex="2"><span class="long-text">about me</span><span class="short-text">am</span></a>
+				<a href="#portfolio" tabindex="2"><span class="long-text">portfolio</span><span class="short-text">p</span></a>
+				<a href="#contact" tabindex="2"><span class="long-text">contact</span><span class="short-text">c</span></a>
 			</div>
 		</nav>
 		<header>
 			<a class="anchor-target" id="about-me"></a>
-			<h1>about me</h1>
+			<h1 tabindex="3">about me</h1>
 			<div>
-				<div>
-					<h2>aspirations</h2>
-					<p>Text about my aspirations to be a software developer ... Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
There may be an overspill of words as your height is fixed.  I guess this just requires editing your copy rather than a coding thing!

Reply:
Thank you - I wasn't looking at this at narrow resolutions!